tests/Infrastructure/Bus/Event/FakeDomainEvent.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CodelyTv\Test\Infrastructure\Bus\Event;
+
+use CodelyTv\Shared\Domain\Bus\Event\DomainEvent;
+
+final class FakeDomainEvent extends DomainEvent
```

Comment:
Refactor into `DomainEventStub#random` named constructor method in order to follow the same pattern as with domain classes